ivy/container/losses.py

```diff
@@ -1,9 +1,106 @@
+# global
+from typing import Optional, Union, List, Dict
+
 # local
+import ivy
 from ivy.container.base import ContainerBase
 
-# ToDo: implement all methods here as public instance methods
-
 
-# noinspection PyMissingConstructor
 class ContainerWithLosses(ContainerBase):
+    def cross_entropy(
```

Comment:
All `ivy.Container` instance methods should make use of the static method internally. Check out the latest [docs on the container](https://github.com/unifyai/ivy/blob/e7712dffabe126fc795d52c2fd0d5b05a91b064f/docs/partial_source/deep_dive/5_containers.rst) for details. The examples given in the container docs are for `add`, `tan` and `roll`.